src/Commands/SetActuator.cpp

```diff
@@ -0,0 +1,39 @@
+#include "SetActuator.h"
+
+SetActuator::SetActuator(Intake::ActuatorValue value)
+: mValue(value)
+{
+	// Use Requires() here to declare subsystem dependencies
+	// eg. Requires(chassis);
```

Comment:
`Requires()`. I won't repeat this comment any longer. Every command should have a requires.